specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2023-12-01-preview/SystemComponents.json

```diff
@@ -0,0 +1,217 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-05-01-preview",
+    "title": "SystemComponentClient",
+    "description": "Use these APIs to create system components through ARM, for Kubernetes Clusters."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{clusterRp}/{clusterResourceName}/{clusterName}/providers/Microsoft.KubernetesConfiguration/systemComponents/default": {
+      "put": {
+        "tags": [
+          "SystemComponents",
+          "ClusterSystemComponents"
+        ],
+        "description": "Create a new Kubernetes Cluster System Components.",
```

Comment:
Is this creating a component? Or is it creating a collection of components? Based on the plural of Components it sounds like it's creating a set of components not just one.

Reply:
+1 to Nikko's comment.  Update the description appropriately.